### DISCUSSION TITLE
Recherche : Ajouter des synomymes pour les expressions courantes

### DISCUSSION THREAD

**Discussion by bobylito** [betagouv/mon-entreprise#2667] [Idées]
Le vocabulaire de la vie courante peut différer de celui des textes administratifs. Pour permettre une meilleure pertinence des résultats, on peut ajouter une liste de synonymes unidirectionnelle.

Exemple :
 - tickets restaurants : titres restaurant
 - chômage partiel : activité partielle

On pourrait utiliser un champ spécifique dans la définition ou utiliser une liste statique qui serait à part.

https://github.com/betagouv/mon-entreprise/pull/1695/files/ad2719785a15cf7dd8ff70dd3a5f651cf4e86502#r704451549

**Comment by johangirod** [comment]
Très bonne idée, je me permet de faire un lien vers une reflexion de @lajarre à ce sujet il y a quelques temps : https://github.com/betagouv/mon-entreprise/pull/964
